File: LLM-Translations/GPT-4/avatar/Python/C/atcoder_ABC142_A.c
#include <stdio.h>

int main() {
    int n, t = 0;
    scanf("%d", &n);
    for(int i = 0; i < n; i++) {
        if((i + 1) & 1) {
            t += 1;
        }
    }
    printf("%f", (float) t / n);
    return 0;
} // End of Code

File: LLM-Translations/GPT-4/avatar/Java/Go/atcoder_ABC142_A.go
package main

import (
	"bufio"
	"fmt"
	"math"
	"math/big"
	"os"
	"strconv"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	input, _ := reader.ReadString('\n')
	inputFloat, _ := strconv.ParseFloat(input, 64)

	countOdd := math.Round(inputFloat / 2)
	result := big.NewRat(int64(countOdd), int64(inputFloat))
	fmt.Println(result.FloatString(10))
}

// //End of Code
